### DISCUSSION TITLE
Install failing on ' pyzmq is not installed'

### DISCUSSION THREAD

**Discussion by olbasoil** [MiczFlor/RPi-Jukebox-RFID#2268] [Q&A] [answered]
I'm  trying to set up my first Phoniebox, and fairly inexperienced in Linux.
Raspberry Pi 3,  fresh install of Bookworm 32 bit
using

>  cd; bash <(wget -qO- https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/future3/main/installation/install-jukebox.sh)
It seems to get quite a long way, before I get 
_ERROR OCCURRED!
A non-recoverable error occurred.
Check install log for details:
/home/pi/INSTALL-1708276161.log
****************************************
ERROR: pyzmq is not installed_

Is there anything I can do to complete this installation? Or should I be using a different Phoniebox/Pi OS?

**Comment by AlvinSchiller** [comment]
Hi, can you provide your full installation log?

**Reply by AlvinSchiller** [reply]
Same issue reported in #2271.
Please post there to track the problem.

**Comment by AlvinSchiller** [comment, marked as answer]
See https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2271.
Will be fixed with https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2272.
